hack/dind-cluster.sh

```diff
@@ -78,7 +78,7 @@ source $(dirname "${BASH_SOURCE}")/../contrib/vagrant/provision-config.sh
 # Enable xtrace for container script invocations if it is enabled
 # for this script.
 BASH_CMD=
-if [ "$(set | grep xtrace)" ]; then
+if [[ "$(set | grep xtrace)" ]]; then
```

Comment:
This can be 
```sh
if set | grep xtrace; then
```

Furthermore if you want to check a specific setting, it's best to do:
```sh
if set +o | grep -q '\-o xtrace'; then
```

or, for the alternative:
```sh
if set +o | grep -q '+o xtrace'; then
```